### DISCUSSION TITLE
Special logit for sampling timestamps

### DISCUSSION THREAD

**Discussion by Majdoddin** [openai/whisper#1075] [Ideas]
Currently, whisper the heuristics " if sum of probability over timestamps is above any other token, sample timestamp" to decide if it is the right place for a timestamp. 
I think it is more natural and efficient to train a special TS logit. If TS has a higher value than all of text logits, then a timestamp is sampled as the next token.
